legend-engine-server/pom.xml

```diff
@@ -351,6 +351,33 @@
             <artifactId>legend-engine-xt-relationalStore-sqlserver-execution</artifactId>
             <scope>runtime</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-relationalStore-spanner-pure</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-relationalStore-spanner-grammar</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-relationalStore-spanner-execution</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-relationalStore-spanner-jdbc-shaded</artifactId>
+            <scope>runtime</scope>
+            <exclusions>
+                <exclusion>
```

Comment:
pls add a comment here describing that we need exclusion to satisfy the enforcer plugin, when shaded dependency is a module within the project

Reply:
Added comment in dependency management, all other exclusions removed